R/parse.R

```diff
@@ -154,35 +154,21 @@ coerce_datetime <- function(x, to, ...) {
 # - "2020-01-01T00:02:03-01:00"
 # nolint end
 parse_connect_rfc3339 <- function(x) {
-  # Convert any timestamps with offsets to a format recognized by `strptime`.
+  # Convert timestamps with offsets to a format recognized by `strptime`.
```

Comment:
Not required, but might be nice to have an example of that the timestamps look like coming in and what they look like after these `gsubs`

Reply:
Good suggestion.